zrml/prediction-markets/src/lib.rs

```diff
@@ -2098,6 +2153,14 @@ mod pallet {
         /// After there was an early close already scheduled,
         /// only the `CloseMarketsEarlyOrigin` can schedule another one.
         OnlyAuthorizedCanScheduleEarlyClose,
+        /// The operation is not allowed for market with a block period.
+        NotAllowedForBlockBasedMarkets,
+        /// The market is not in the open time frame list.
+        MarketNotInOpenTimeFrameList,
```

Comment:
We don't need all of these.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1160/commits/425edd2dede1029d26228c4dc79329c5de24d240